exporter.go

```diff
@@ -110,6 +113,42 @@ func createClientWithRetries(getClient func() (interface{}, error), retries uint
 	return nil, err
 }
 
+func parseUnixSocketAddress(listenAddress string) (string, string, error) {
```

Comment:
since this function is used not only for parsing a listening unix socket but also for the one to scape the metrics from, I suggest dropping the prefix listen from variable names  in this function.

Reply:
I added the unit tests and removed all listen prefix.